web/templates/message_controls.hbs

```diff
@@ -1,36 +1,19 @@
-<div class="message_controls no-select">
-    {{#if msg/sent_by_me}}
+{{#if msg/sent_by_me}}
     <div class="edit_content message_control_button"></div>
-    {{/if}}
+{{/if}}
 
-    {{#unless msg/sent_by_me}}
+{{#unless msg/sent_by_me}}
     <div class="reaction_button message_control_button" data-tooltip-template-id="add-emoji-tooltip-template">
         <div class="emoji-message-control-button-container">
             <i class="message-controls-icon zulip-icon zulip-icon-smile" aria-label="{{t 'Add emoji reaction' }} (:)" role="button" aria-haspopup="true" tabindex="0"></i>
         </div>
     </div>
-    {{/unless}}
+{{/unless}}
 
-    {{#unless msg/locally_echoed}}
```

Comment:
There's no need for checking the `locally_echoed` flag here since we check those in `web/templates/message_body.hbs` now.